category/domain/category.go

```diff
@@ -25,10 +25,18 @@ type (
 		CategoryMedia      Medias
 		CategoryTypeCode   string
 		CategoryAttributes Attributes
+		Promotion          Promotion
 	}
 
 	// Attributes define additional category attributes
 	Attributes map[string]interface{}
+
+	// Promotion defines promotion for a category
+	Promotion struct {
```

Comment:
Why is this part of this PR?

Reply:
There was a mistake while rebasing. Will close this merge request and open a new one. Will also copy over your other comments.